fiftyone/core/fields.py

```diff
@@ -177,14 +178,27 @@ def __str__(self):
 
         return etau.get_class_name(self)
 
+    def validate(self, value):
+        if isinstance(self.field, EmbeddedDocumentField):
+            for v in value:
+                self.field.validate(v, clean=False, expand=True)
```

Comment:
Validate every document in a `ListField` with an `EmbeddedDocumentField` field. A pain point for performance, likely.